bundle/schema/docs/bundle_descriptions.json

```diff
@@ -182,7 +182,7 @@
                 "description": "An optional continuous property for this job. The continuous property will ensure that there is always one run executing. Only one of `schedule` and `continuous` can be used.",
                 "properties": {
                   "pause_status": {
-                    "description": "Indicate whether the continuous execution of the job is paused or not. Defaults to UNPAUSED."
+                    "description": "Whether this trigger is paused or not."
```

Comment:
Can you regen with the latest version of the deco CLI? I think these mismatches are due to some changes in the OpenAPI processing code.

Reply:
done